examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor_postprocess.py

```diff
@@ -11,19 +11,59 @@
 # Modules
 #-------------------------------------------
 import numpy as np
+import argparse
 import pandas as pd
 import matplotlib.pyplot as plt
+import matplotlib.patheffects as pe
 import pyvista as pv
+import tqdm
 
 import os
 import sys
+sys.path.append("$LETHE_PATH/contrib/postprocessing/")
+from lethe_pyvista_tools import *
+
+
+#--------------------------------------------
+# Plot parameters
+#--------------------------------------------
+from cycler import cycler
+
+colors=['#1b9e77','#d95f02','#7570b3','#e7298a','#66a61e','#e6ab02']
+
+plt.rcParams['axes.prop_cycle'] = cycler(color = colors)
+plt.rcParams['figure.facecolor'] = 'white'
+plt.rcParams['figure.figsize'] = (10,8)
+plt.rcParams['lines.linewidth'] = 4
+plt.rcParams['lines.markersize'] = '11'
+plt.rcParams['markers.fillstyle'] = "none"
+plt.rcParams['lines.markeredgewidth'] = 2
+plt.rcParams['legend.columnspacing'] = 2
+plt.rcParams['legend.handlelength'] = 3
+plt.rcParams['legend.handletextpad'] = 0.2
+plt.rcParams['legend.frameon'] = True
+plt.rcParams['legend.fancybox'] = False
+plt.rcParams['xtick.major.width'] = 2
+plt.rcParams['xtick.major.size'] = 5
+plt.rcParams['ytick.major.size'] = 5
+plt.rcParams['ytick.major.width'] = 2
+plt.rcParams['font.size'] = '25'
+plt.rcParams['font.family']='DejaVu Serif'
+plt.rcParams['font.serif']='cm'
+plt.rcParams['savefig.bbox']='tight'
+plt.rcParams['legend.handlelength']=1
```

Comment:
This changes the font (to latex) and allows also the italicized writing for "_et al._" in the figure legends.
Figures will need to be updated also if the changes are applied.

```suggestion
plt.rcParams['legend.handlelength']=1
plt.rcParams['text.usetex'] = True
```